### ISSUE TITLE
Manually clear a throttle

### ISSUE THREAD

**Issue by lowerends** [GrahamCampbell/Laravel-Throttle#42]
Is there a way to manually clear a throttle if a user has reached the maximum number of attempts, when using an instance of `\Illuminate\Http\Request` as the first parameter for creating a throttler class and using the file cache driver?

**Comment by GrahamCampbell** [comment]
https://github.com/GrahamCampbell/Laravel-Throttle/blob/master/src/Throttlers/ThrottlerInterface.php#L35-L40

**Comment by lowerends** [comment]
Yes, I'm aware of the `clear` method, but with that method there's no way of knowing which request to clear is it?

Let me rephrase my question a bit to indicate what I mean with _manually_ clearing a throttle: is there a way to clear any or all throttles using tinker or a custom artisan command?

**Comment by GrahamCampbell** [comment]
That method DOES know which request to clear.

**Comment by lowerends** [comment]
You're right, but when I want to call that method manually using tinker or in a custom artisan command, how do I know which request to clear? 

Is there a way to clear all throttles?

**Comment by GrahamCampbell** [comment]
> how do I know which request to clear?

Construct an array rather than a request. Check out the transformer stuff so you can work out how to minic it correctly.

**Comment by lowerends** [comment]
Ok, thanks.
